### DISCUSSION TITLE
Secure WebSocket connections as a client, how to manage certificates?

### DISCUSSION THREAD

**Discussion by Style12341** [Links2004/arduinoWebSockets#920] [Q&A]
I have a web server that has a web socket endpoint. This web server is behind a reverse proxy that redirects requests on port 80 to 443 to be managed by https and certificates are managed by Let's Encrypt.

The main question is, why can't we have here something similar as the client.setInsecure() on http, which enables https connections without certificate verification. Without this I am obligated to provide server certificates regularly or install a bundle which will also expire someday. This is bad user experience if i'm developing a library.

TL:DR How are certificates managed in real applications that don't require user intervention?
Would it be possible for a setInsecure() function to exist in this library?
